onyo/lib/utils.py

```diff
@@ -201,7 +201,7 @@ def dict_to_asset_yaml(d: Dict | UserDict) -> str:
     return s.getvalue()
 
 
-def get_asset_content(asset_file: Path) -> dict[str, bool | float | int | str | Path]:
+def get_asset_content(asset_file: Path) -> dict:
```

Comment:
In this case, I think we do want to specify the type of the key, since that's literally what we're doing. What about `dict[str, Any]:`

Reply:
Won't fix. Clarified in chat.